spec/system/support/providers/manual_rollover_provider_spec.rb

```diff
@@ -0,0 +1,85 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+RSpec.describe "Manual provider rollover", service: :publish do
+  include DfESignInUserHelper
+
+  let(:provider) { create(:provider, recruitment_cycle: current_recruitment_cycle) }
+  let(:user) { create(:user, :admin, providers: [provider]) }
+  let!(:current_recruitment_cycle) { RecruitmentCycle.current }
+  let!(:next_recruitment_cycle) { create(:recruitment_cycle, :next) }
+
+  before do
+    Timecop.travel(Time.zone.local(2025, 1, 1))
```

Comment:
```suggestion
    Timecop.travel(Find::CycleTimetable.mid_cycle)
```

The year 2025 won't make sense in a year when we use `RecruitmentCycle.current`